index.js

```diff
@@ -288,7 +288,7 @@ function loadConfig(program) {
     }
 
     debug('connecting to log server')
-    connectionManager.connectLogServer(config)
+    createLoggerThenStart(config)
```

Comment:
can we just name it as `start`? it seems like `createLoggerThenStart` indicates that there is another `...Start` method in which no logger creation.

I mean, we already know that it won't start anything without creating logger - so, `start(config)` would be better, imo

Reply:
I'll buy that.  Naming is hard.

Reply:
done